attributes/default.rb

```diff
@@ -225,3 +225,4 @@
 
 # attribute to add commands to source build
 default['nagios']['source']['add_build_commands'] = ['make install-exfoliation']
+default['nagios']['alloved_ips'] = []
```

Comment:
@mdarii I assume this is a typo, and should be "allowed_ips" :) Can you fix it here and below